src/Discord.Net.Core/Entities/Permissions/GuildPermission.cs

```diff
@@ -192,17 +192,25 @@ public enum GuildPermission : ulong
         /// </remarks>
         ManageThreads           = 0x04_00_00_00_00,
         /// <summary>
-        /// Allows for creating and participating in threads.
+        ///     Allows for creating public threads.
         /// </summary>
-        UsePublicThreads        = 0x08_00_00_00_00,
```

Comment:
Could you add back `UsePublicThreads` and `UsePrivateThreads` with the `[Obsolete]` attribute explaining the rename?

Reply:
Added back to GuildPermission.cs, with [Obsolete], directing to Create...Threads and SendMessagesInThreads.